README.md

```diff
@@ -79,6 +79,28 @@ set :delayed_job_queues, ['mailer','tracking']
 #     'high_priority,*,low_priority' => 1, # works on whatever is available
 #     '*,low_priority' => 1,  # high_priority doesn't starve the little guys
 #   }
+
+# Several workers each handling one or more queues, different settings 
+# per machine:
+# 
+# set :delayed_job_pools_per_machine, {
+#   'machine-1' => {
+#     'default,emails' => 3,
+#     'loud_notifications' => 1,
+#     'silent_notifications' => 1,
+#   },
+#   'machine-2' => {
+#     'default' => 2
+#   }
+# }
+
+# set :delayed_job_pools, {
```

Comment:
Did you intend to duplicate these settings in the docs?
It seems like one would use `:delayed_job_pools_per_machine` in preference to `:delayed_job_pools` rather than use both. Was that not your expectation?

Reply:
Yes, I wanted to duplicate the example for `set :delayed_job_pools_per_machine` from `set :delayed_job_pools`. 
I will deduplicate it in the next PR. 
Corrected version is here: 
https://github.com/goodylabs/capistrano3-delayed-job/commit/8f3b5cb768d15ddf0d58614504f1d9f25c828d0c